Sources/UIView+Sweetness.swift

```diff
@@ -20,6 +20,15 @@ public extension UIView {
         self.init()
         self.translatesAutoresizingMaskIntoConstraints = !autoLayout
     }
+
+    public func fillSuperview(with insets: UIEdgeInsets) {
```

Comment:
Could we make the insets optional? So you could easily just set the exact same bounds? Or maybe create another method for that.